dist/dable.js

```diff
@@ -0,0 +1,608 @@
+/*! dable v1.2.1 (https://github.com/deltreey/Dable) */
+(function() {
+    var $$modules = {}, defineModule = function(name, exporter) {
+        $$modules[name] = {
+            exporter: exporter,
+            ready: !1
+        };
+    }, require = function(name) {
+        var m = $$modules[name];
+        return m && !m.ready && (m.exports = {}, m.exporter.call(null, require, m, m.exports), 
+        m.ready = !0), m && m.exports;
+    };
+    defineModule("./utils", function(require, module, exports) {
+        exports.mixin = function(obj, mixins) {
+            for (var keys = Object.keys(mixins), i = 0; i < keys.length; i++) {
+                var key = keys[i];
+                obj[key] = mixins[key];
+            }
+        }, exports.bind = function(fn, thisArg) {
+            return function() {
+                var args = Array.prototype.slice.call(arguments, 0);
+                return fn.apply(thisArg, args);
+            };
+        }, exports.noop = function() {};
+    }), function(module) {
+        function RemoveStyle(node) {
+            node.removeAttribute("style");
+            var childNodes = node.children;
+            if (childNodes && childNodes.length > 0) for (var i = 0; i < childNodes.length; ++i) RemoveStyle(childNodes[i]);
```

Comment:
Line is too long.<br>Expected '{' and instead saw 'for'.<br>Expected '{' and instead saw 'RemoveStyle'.<br>Missing 'new' prefix when invoking a constructor.